src/AppBundle/Api/ApiClient.php

```diff
@@ -367,6 +389,41 @@ public function findGroups(
     }
 
     /**
+     * @param int $groupId
+     * @param string $query
+     * @param string $type
+     * @param int $offset
+     * @param int $limit
+     * @param SortOrder $sortOrder
+     * @param int[] $groupIds
+     * @return Collection
+     */
+    public function findGroupsLinkable(
+        $groupId,
+        $query,
+        $type,
+        $offset,
+        $limit,
+        SortOrder $sortOrder,
+        array $groupIds = null
+    ) {
+        $data = $this->guzzle->get(sprintf('groups/%u/groups/linkable', $groupId), [
+            'query' => [
+                'offset' => $offset,
+                'limit'  => $limit,
+                'sort'   => $sortOrder->toSignedOrder(),
+                'type'   => $type,
+                'query'  => $query,
+                'ids'    => $groupIds
+            ],
+        ]);
+
+        $data = $this->decode($data->getBody());
```

Comment:
Some exception handling would be nice. This way, one could be more informative about what went wrong.